File: 1560-most-visited-sector-in-a-circular-track/1560-most-visited-sector-in-a-circular-track.py
class Solution:
    def mostVisited(self, n: int, rounds: List[int]) -> List[int]:
        
        
        start = rounds[0]
        end = rounds[-1]
        
        
        if start <= end:
            return [i for i in range(start, end+1)]
        
        else:
            
            res = []
            
            for i in range(n):
                
                if i + 1 <= end or i + 1 >= start:
                    res.append(i+1)
                    
            return res

File: 1560-most-visited-sector-in-a-circular-track/1560-most-visited-sector-in-a-circular-track.js
/**
 * @param {number} n
 * @param {number[]} rounds
 * @return {number[]}
 */
var mostVisited = function(n, rounds) {
    let first = rounds[0]
    let last = rounds[rounds.length -1]
    let res = []
    if (first <= last){
        for(let i = first; i < last+1; i++){
            res.push(i)
        }
        
    } else{
        
        for (let i = 0; i<n; i++){
            if (i + 1 <= last || i+1 >= first){
                res.push(i+1)
            }
        }
    }
    return res
};